panther_docking/package.xml

```diff
@@ -20,7 +20,9 @@
   <depend>opennav_docking</depend>
   <depend>panther_utils</depend>
   <depend>pluginlib</depend>
+  <depend>python3-imageio</depend>
```

Comment:
Seems like an exec dependency.

Reply:
Do we want to separate this?

Reply:
There are no strict guidelines for specifying dependencies in this project. IMO it's worth doing that but in the end, it's up to you. You would revise all deps in this particular package.xml file.

Reply:
Moved to exec_depend